SPECS/ocaml-ppx-let/ocaml-ppx-let.spec

```diff
@@ -0,0 +1,139 @@
+# OCaml packages not built on i686 since OCaml 5 / Fedora 39.
+ExcludeArch: %{ix86}
+
+Name:           ocaml-ppx-let
+Version:        0.16.0
+Release:        9%{?dist}
+Summary:        Monadic let-bindings for OCaml
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/janestreet/ppx_let
+Source0:        %{url}/archive/v%{version}/ppx_let-%{version}.tar.gz
+BuildRequires:  ocaml >= 5.1.1
+BuildRequires:  ocaml-base-devel >= 0.16
+BuildRequires:  ocaml-dune >= 2.0.0
+BuildRequires:  ocaml-ppx-here-devel >= 0.16
+BuildRequires:  ocaml-ppxlib-devel >= 0.31.0
+
+%description
+Ppx_let is a ppx rewriter for monadic and applicative let bindings,
+match expressions, and if expressions.
+
+The aim of this rewriter is to make monadic and applicative code look
+nicer by writing custom binders the same way that we normally bind
+variables.  In OCaml, the common way to bind the result of a computation
+to a variable is:
+
+  let VAR = EXPR in BODY
+
+ppx_let simply adds two new binders: let%%bind and let%%map.  These are
+rewritten into calls to the bind and map functions respectively.  These
+functions are expected to have
+
+  val map  : 'a t -> f:('a -> 'b)   -> 'b t
+  val bind : 'a t -> f:('a -> 'b t) -> 'b t
+
+for some type t, as one might expect.
+
+These functions are to be provided by the user, and are generally
+expected to be part of the signatures of monads and applicatives
+modules.  This is the case for all monads and applicatives defined by
+the Jane Street's Core suite of libraries.
+
+%package        devel
+Summary:        Development files for %{name}
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+Requires:       ocaml-base-devel%{?_isa}
+Requires:       ocaml-ppx-here-devel%{?_isa}
+Requires:       ocaml-ppxlib-devel%{?_isa}
+
+%description    devel
+The %{name}-devel package contains libraries and signature
+files for developing applications that use %{name}.
+
+%prep
+%autosetup -n ppx_let-%{version}
+
+%build
+%dune_build
+
+%install
+%dune_install
+
+%check
+%dune_check
+
+%files -f .ofiles
+%doc CHANGES.md README.md
+%license LICENSE.md
+
+%files devel -f .ofiles-devel
+
+%changelog
+* Wed May 01 2024 Mykhailo Bykhovtsev <mbykhovtsev@microsoft.com> - 0.16.0-9
```

Comment:
Missed release bump. Please check all new specs, I think this may be common across them.